### DISCUSSION TITLE
Alternative Icons

### DISCUSSION THREAD

**Discussion by jaidetree** [kovidgoyal/kitty#6107] [Show and tell]
Hey there! Recently switched from Alacritty and found I really prefer Kitty. The visual concept is the original icon and logo is very fun but wanted to take it further.

Personally, I really wanted a 3D version of the original design, but after chatting with folks about it in the [art-design channel in the doom-emacs discord server](https://discord.com/channels/406534637242810369/1079089639652085912), they had another direction in mind that also seemed fun to explore.

![kitty 2d](https://user-images.githubusercontent.com/590297/224456826-9519db7e-1b2f-4e44-a208-cf1132e50b61.png)

![kitty 2d preview](https://user-images.githubusercontent.com/590297/224457201-607d3bf2-c8e7-45f7-9fd7-ac185a6d1bd4.png)

![kitty 3d](https://user-images.githubusercontent.com/590297/224456845-ef425ab2-1e09-43ae-873e-266fabdddaf9.png)

![kitty 3d preview](https://user-images.githubusercontent.com/590297/224457164-231df93e-923d-4b34-8808-e30e834730a2.png)

## Licensing

They are licensed under [CC0](https://github.com/eccentric-j/eccentric-icons/blob/0cc136173605598c69327154d49e944c126ccd41/LICENSE) so they're free to use as people please. Credit is appreciated if repackaged elsewhere.

## Download Formats

Working on automating the format generation but have MacOS .icns and png formats available at:
https://github.com/eccentric-j/eccentric-icons/tree/main/icons/kitty-terminal

## Credit

I would like to thank Hylo, PersonaNull, Hi_Im_Bored, and Lunik1 in the doom-emacs discord server for all their help brainstorming, providing feedback, and suggestions to get these to a more polished state.

## Changes

Feel free to request changes. The 3d one especially is only the 4th character design I've ever done so feels like still a lot to learn. I can also share my source files as well.

**Comment by kovidgoyal** [comment]
Cool, I like the winky one. Can you make a PR to add your icons to the docs/faq.rst so others can find them easily.

**Reply by jaidetree** [reply]
Yeah I can do that.

**Comment by kovidgoyal** [comment]
Also changing to a discussion as its not really an issue.

**Reply by jaidetree** [reply]
Fair point, realized after I should have created a discussion.
